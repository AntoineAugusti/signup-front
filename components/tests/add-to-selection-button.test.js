import ReactTestRenderer from 'react-test-renderer'
import AddToSelectionButton from '../add-to-selection-button'

describe('components | AddToSelectionButton', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<AddToSelectionButton />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
