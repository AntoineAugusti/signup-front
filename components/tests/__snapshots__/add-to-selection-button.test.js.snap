// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | AddToSelectionButton render should match snapshot 1`] = `
<a
  href="/contractualisation"
  onClick={[Function]}
>
  Ajouter à ma sélection de jeux de données !
</a>
`;
